lib/modules/datasource/azure-tags/index.ts

```diff
@@ -0,0 +1,49 @@
+import { cache } from '../../../util/cache/package/decorator';
+import type { PackageCacheNamespace } from '../../../util/cache/package/namespaces';
+import { ensureTrailingSlash } from '../../../util/url';
+import * as azureHelper from '../../platform/azure/azure-helper';
+import { Datasource } from '../datasource';
+import type { GetReleasesConfig, ReleaseResult } from '../types';
+
+export class AzureTagsDatasource extends Datasource {
+  static readonly id = 'azure-tags';
+
+  static readonly cacheNamespace: PackageCacheNamespace = `datasource-${AzureTagsDatasource.id}`;
+
+  constructor() {
+    super(AzureTagsDatasource.id);
+  }
+
+  static getCacheKey(registryUrl: string, repo: string, type: string): string {
+    return `${registryUrl}:${repo}:${type}`;
+  }
+
+  static getSourceUrl(packageName: string, registryUrl: string): string {
+    const normalizedUrl = ensureTrailingSlash(registryUrl);
+    return `${normalizedUrl}_git/${packageName}`;
+  }
+
+  @cache({
+    namespace: AzureTagsDatasource.cacheNamespace,
+    key: ({ registryUrl, packageName }: GetReleasesConfig) =>
+      AzureTagsDatasource.getCacheKey(registryUrl!, packageName, 'tags'),
+  })
+  async getReleases({
+    registryUrl,
+    packageName: repo,
+  }: GetReleasesConfig): Promise<ReleaseResult | null> {
+    const azureTags = await azureHelper.getTags(repo);
```

Comment:
I think we need to use `http.getJson` here..importing from `platform` module might be undesirable.

@viceice wdyt?